SU-SWS/decanter-react
# Adapt-1955 Adapt-1700 Adapt 1960 Logo Lockup, Local Footer, Heading components

# READY FOR REVIEW

# Summary
- New components: Lockup, Local Footer, Heading
- Tweaks to FlexCell and GridCell. FlexCell can now specify width for each breakpoint; GridCell can now specific column span for each breakpoint.
- Add unlink option to Stanford Logo component

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Look at preview build and check the example stories and play with the controls
1. Look at code

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
